backend-lucene/src/main/java/org/hibernate/search/v6poc/backend/lucene/logging/impl/Log.java

```diff
@@ -183,4 +183,11 @@ SearchException conflictingFieldTypesForSearch(String absoluteFieldPath,
 
 	@Message(id = 543, value = "Descending order is not supported by distance sort for field '%1$s'.")
 	SearchException descendingOrderNotSupportedByDistanceSort(String absoluteFieldPath);
+
+	@Message(id = 544, value = "Computed minimum for minimumShouldMatch constraint was out of bounds:"
+			+ " expected a number between 1 and '%1$s', got '%2$s'.")
+	SearchException minimumShouldMatchMinimumOutOfBounds(int minimum, int totalShouldClauseNumber);
+
+	@Message(id = 545, value = "Multiple conflicting minimumShouldMatch constraints for ceiling '%1$s'")
+	SearchException minimumShouldMatchConflictingConstraints(int ceiling);
```

Comment:
And here we have `ceiling` again. So my suggestion for the parameter name might not be ideal.